### DISCUSSION TITLE
Does this adapter support the SYR Safe-T Connect

### DISCUSSION THREAD

**Discussion by pustelbaer** [smarthausleben/ioBroker.wamo#17] [Q&A] [answered]
the SYR Safe-T Connect 4221 seems to be very similar to the SYR Safe-Tech 2422.
Does the adapter support this device as well?

**Comment by smarthausleben** [comment]
I don’t think so. But I can’t tell you for sure. Just give it a try, maybe it works.

Unfortunately my contact to SYR is broken because I can’t reach my contact person there anymore and therefore I have no way to get the documentation for your device. As soon as I have reestablished the communication SYR, I will try to get API documentation for other devices to implement them into the adapter.

If you have some API documentation for your device, please let me know.

**Comment by pustelbaer** [comment]
Thanks for the info. I'll try to get the API documentation for the Safe-T Connect from SYR.

**Comment by pustelbaer** [comment, marked as answer]
For all those who are faced with the same question: According to information from SYR, the Safe-T Connect has no API interface, so it is not suitable. I have now had the SafeTech Plus Connect installed, which supposedly has the same circuit board and firmware as the SafeTech Connect. It works perfectly with the adapter.
